src/NLog/Targets/ColoredConsoleAnsiPrinter.cs

```diff
@@ -51,7 +51,7 @@ public TextWriter AcquireTextWriter(TextWriter consoleStream, StringBuilder reus
             return new StringWriter(reusableBuilder ?? new StringBuilder(50), consoleStream.FormatProvider);
         }
 
-        public void ReleaseTextWriter(TextWriter consoleWriter, TextWriter consoleStream, ConsoleColor? oldForegroundColor, ConsoleColor? oldBackgroundColor, bool flush)
+        public void ReleaseTextWriter(TextWriter consoleWriter, TextWriter consoleStream, ConsoleColor? orgForegroundColor, ConsoleColor? orgBackgroundColor, bool flush)
```

Comment:
please write out org, prev, etc

Reply:
Uhh suddenly old is good again. Will change to old then